applications/swbatch/app.json

```diff
@@ -0,0 +1,219 @@
+                }
+              },
+              {
+                "key": "Number of frequency points in exported dispersion curve(s)",
+                "value": "30",
+                "description": "Number (positive integer) of frequency points in the exported dispersion curve(s). (30 is recommended)",
+                "inputMode": "REQUIRED",
+                "notes": {
+                  "label": "dcfnum"
+                }
+              },
+              {
+                "key": "Number of Rayleigh modes to include in exported ellipticity",
+                "value": "1",
+                "description": "Number (positive integer) of Rayleigh modes to include in exported ellipticity. If no ellipticity curves are desired set this value to 0. (1 is recommended)",
+                "inputMode": "REQUIRED",
+                "notes": {
+                  "label": "nellipticity"
+                }
+              },
+              {
+                "key": "Minimum frequency of exported Rayleigh wave ellipticity in Hz",
+                "value": "",
+                "description": "Number (positive float) for minimum frequency of exported Rayleigh wave ellipticity curve(s) in Hz. Selecting a value less than the site's resonant frequency is recommended.",
+                "inputMode": "REQUIRED",
+                "notes": {
+                  "label": "ellfmin"
+                }
+              },
+              {
+                "key": "Maximum frequency of exported Rayleigh wave ellipticity in Hz",
+                "value": "",
+                "description": "Number (positive float) for maximum frequency of exported Rayleigh wave ellipticity curve(s) in Hz. Selecting a value greater than the site's resonant frequency is recommended.",
+                "inputMode": "REQUIRED",
+                "notes": {
+                  "label": "ellfmax"
+                }
+              },
+              {
+                "key": "Number of frequency points in exported Rayleigh wave ellipticity curve(s)",
+                "value": "64",
+                "description": "Number (positive integer) of frequency points in exported Rayleigh wave ellipticity curve(s). (64 is recommended)",
+                "inputMode": "REQUIRED",
+                "notes": {
+                  "label": "ellfnum"
+                }
+              }
+            ]        
+        },
+        "fileInputs": [
+            {
+                "name": "Working Directory",
+                "description": "The directory containing the target (i.e., 0_targets) and parameter (i.e., 1_parameters) directories, see documentation for details. If using the portal interface (i.e., not using Jupyter) you can drag the link for the directory from the Data Depot Browser on the left, or click the 'Select' button to select the desired directory.",                
+                "inputMode": "REQUIRED",
+                "autoMountLocal": true,
+                "sourceUrl": null,
```

Comment:
Do we have any example input for this app? If not, we should reach out to Joe V, who I think is still on slack.

Reply:
I found an example file here: https://github.com/jpvantassel/swbatch?tab=readme-ov-file